tests/sqllogictests/suites/base/01_system/01_0013_system_metrics

```diff
@@ -0,0 +1,16 @@
+onlyif mysql
+query T
+SELECT metric FROM system.metrics where metric like '%mysql_interpreter_usedtime_ms_count%' and to_float32(value) > 5.0
+----
+mysql_interpreter_usedtime_ms_count
+
+onlyif mysql
+statement ok
+----
+truncate table system.metrics;
+
+onlyif mysql
+query TI
+SELECT metric, value FROM system.metrics where metric like '%mysql_interpreter_usedtime_ms_count%'
+----
+mysql_interpreter_usedtime_ms_count 2.0
```

Comment:
This result is not as expected?
For test, it should be: `mysql_interpreter_usedtime_ms_count 0.0`

Reply:
Or choose another metric to check?

Reply:
I guess this metrics got affected with the `TRUNCATE TABLE` itself, it will increment when the statement itself finished.

I choosed another metric `query_scan_bytes_total` in my later commit

Reply:
I saw the new one.
But before check `query_scan_bytes_total ` we should do some select make `query_scan_bytes_total ` is not zero.

Reply:
`select sum(number) from numbers(10);` will do that.